utils/srg_diff.html

```diff
@@ -28,6 +28,24 @@ <h2>Missing in CaC</h2>
         <li>{{ rule }}</li>
     {% endfor %}
 </ul>
+{% if disa_missing_cces %}
+<h2>Requirements Missing CCE in DISA</h2>
+<ul>
+    {% for requirement in disa_missing_cces %}
+        <li>{{ requirement }}</li>
+    {% endfor %}
+</ul>
+{% endif %}
+
+{% if cac_missing_cces %}
+<h2>Requirements Missing CCE in CaC</h2>
+<ul>
+    {% for requirement in cac_missing_cces %}
+        <li>{{ requirement }}</li>
```

Comment:
Would it be useful to have here also the SRG ID to be able to find the control file or spreadsheet row easier?